lang/luajit2/Makefile

```diff
@@ -0,0 +1,93 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luajit2
+PKG_VERSION:=2.1-20230410
+PKG_RELEASE:=1
+
+PKG_SOURCE:=luajit2-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/openresty/luajit2/tar.gz/v$(PKG_VERSION)?
```

Comment:
please no codeload just use the github repository, we had some funny surprise with codeload generated tar (with the changing compression and hash changing)